docs/json-examples.md

```diff
@@ -0,0 +1,744 @@
+  "extra": {
+    "valueLost": 300,
+    "isPvp": false,
+    "killerName": "%NPC%",
+    "killerNpcId": 69,
+    "keptItems": [],
+    "lostItems": [
+      {
+        "id": 314,
+        "quantity": 100,
+        "priceEach": 3,
+        "name": "Feather"
+      }
+    ]
+  },
+  "type": "DEATH"
+}
+```
+
+### Collection
+
+JSON for Collection Notifications:
+
+```json5
+{
+  "content": "%USERNAME% has added %ITEM% to their collection",
+  "extra": {
+    "itemName": "Zamorak chaps",
+    "itemId": 10372,
+    "price": 500812,
+    "completedEntries": 420,
+    "totalEntries": 1443
+  },
+  "type": "COLLECTION"
+}
+```
+
+### Level
+
+JSON for Levelups:
+
+```json5
+{
+  "content": "%USERNAME% has levelled %SKILL%",
+  "extra": {
+    "levelledSkills": {
+      // These are the skills that dinked
+      "Skill name": 30
+    },
+    "allSkills": {
+      // These are all the skills
+      "Skill name": 30,
+      "Other skill": 1
+    },
+    "combatLevel": {
+      "value": 50,
+      "increased": false
+    }
+  },
+  "type": "LEVEL"
+}
+```
+
+Note: Level 127 in JSON corresponds to attaining max experience in a skill (200M).
+
+### Loot
+
+JSON for Loot Notifications:
+
+```json5
+{
+  "content": "%USERNAME% has looted: \n\n%LOOT%\nFrom: %SOURCE%",
+  "extra": {
+    "items": [
+      {
+        // type of this object is SerializedItemStack
+
+        "id": 1234,
+        "quantity": 1,
+        "priceEach": 42069,
+        // priceEach is the GE price of the item
+        "name": "Some item"
+      }
+    ],
+    "source": "Giant rat",
+    "category": "NPC",
+    "killCount": 60
+  },
+  "type": "LOOT"
+}
+```
+
+`killCount` is only specified for NPC loot with the base RuneLite Loot Tracker plugin enabled.
+
+### Slayer
+
+JSON for Slayer Notifications:
+
+```json5
+{
+  "content": "%USERNAME% has completed a slayer task: %TASK%, getting %POINTS% points and making that %TASKCOUNT% tasks completed",
+  "extra": {
+    "slayerTask": "Slayer task name",
+    "slayerCompleted": "30",
+    "slayerPoints": "15",
+    "killCount": 135,
+    "monster": "Kalphite"
+  },
+  "type": "SLAYER"
+}
+```
+
+### Quests
+
+JSON for Quest Notifications:
+
+```json5
+{
+  "content": "%USERNAME% has completed a quest: %QUEST%",
+  "extra": {
+    "questName": "Dragon Slayer I",
+    "completedQuests": 22,
+    "totalQuests": 156,
+    "questPoints": 44,
+    "totalQuestPoints": 293
+  },
+  "type": "QUEST"
+}
+```
+
+### Clue
+
+JSON for Clue Notifications:
+
+```json5
+{
+  "content": "%USERNAME% has completed a %CLUE% clue, they have completed %COUNT%.\nThey obtained:\n\n%LOOT%",
+  "extra": {
+    "clueType": "Beginner",
+    "numberCompleted": 123,
+    "items": [
+      {
+        // the type of this object SerializedItemStack
+
+        "id": 1234,
+        "quantity": 1,
+        "priceEach": 42069,
+        // priceEach is the GE price of the item
+        "name": "Some item"
+      }
+    ]
+  },
+  "type": "CLUE"
+}
+```
+
```

Comment:
```suggestion

The items are valued at GE prices (when possible) if the user has not disabled the `Use actively traded price` base RuneLite setting. Otherwise, the store price of the item is used.

```